src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
Please be more specific, it's not a random string... an enum would be a great solution for this

Reply:
done)